cmd/ethereum_rust/ethereum_rust.rs

```diff
@@ -111,17 +113,31 @@ async fn main() {
         node_id: local_node_id,
     };
 
+    // TODO: Check every module starts properly.
```

Comment:
Is this about other modules also tracking their tasks or a different thing? How does the tracker propagate, given it's not passed as an argument?

Reply:
The tracker now only spawns the workers but does not check they start properly, maybe each module/process should return if they could start or not and give them a couple of retries, maybe this should be an issue, wdy @mpaulucci ?

Reply:
I think it merits an issue to track that down, yes.